.github/workflows/main.yaml

```diff
@@ -0,0 +1,34 @@
+name: Main branch CI
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
How about removing the branch from the PR trigger? Is there any reason why we wouldn't want the CI to run for PRs to other branches, like release ones, and similar?
```suggestion
```

Reply:
Once we have a release branch we can add it to the CI.
There was a discussion around this, and I remember the rationale behind it was that you might not want to trigger the whole ci just by pushing some changes to a dev branch.